### ISSUE TITLE
Spawn in arena after dying in other world

### ISSUE THREAD

**Issue by Seba244c** [ScreamingSandals/BedWars#192]
#### Describe the bug
When someone right clicks on one of the beds in an Arena it sets their spawn there, sp that every time they die EVEN in other worlds. They get teleported back to the arena. 

#### Server and plugin version
Spigot version: Newest buildtool version
BedWars version: 0.2.10

#### Installed plugins on server (can be obtained by `/pl`)
RemoveLeaveJoinMessages, WorldEdit, Voidgenerator, GroupManager, PlacegolderAPI, BuycraftX, Vault, Multiverse-Core, ProtocolLib, WORLDGUARD, NoCheatplus, Essentials, EssentialsProtect, EssentialsChat, EssentialsAntiBuild, BEDWARS, HolographicDisplays, Multiverse Portals

#### Console log
None

#### Configuration (of config.yml and shop.yml)
Shop only changed the price of arrows.

Config.yml: https://pastebin.com/6TVq01aw

#### If there is problem with only one game, paste here config of this game
I only have on game cause I am still testing this plugin: https://pastebin.com/ZTNtpDdW

**Comment by iamceph** [comment]
You did not provided your Minecraft version

**Comment by Seba244c** [comment]
1.15.2

**Comment by Misat11** [comment]
Try latest dev build of BedWars and check if this issue is still here or not.

**Comment by Seba244c** [comment]
Ok updated bedwars to 0.2.11-SNAPSHOT-553, from Jenkins and the issue still persists.

**Comment by Misat11** [comment]
So you can still set your bed, right?

**Comment by Seba244c** [comment]
WDYM set your bed?

**Comment by Misat11** [comment]
when you click on it with the right mouse button and then it's selected as respawn point

**Comment by Seba244c** [comment]
Yes that's the problem. I don't want my players to set their spawn in the Arena.

**Comment by Misat11** [comment]
With Multiverse, you can disable bedRespawn for specific worlds, so you can temporary fix your issue with it.

**Comment by AtherActive** [comment]
If you ask me this is more a setup issue. If players can full on die-die which is outside of the game mode then you have not set up your lobby well. Now I understand that having multiple game modes inside one server seems to work fine, but it is a headache. I recommend using Bungeecord and just splitting it over two servers (if you have that choice)

What i recommend is to have something that will ALWAYS TP a player back to spawn (on join, death etc). There are some good plugins for that. Also, like said before, split the load.
